addons/purchase/views/assets.xml

```diff
@@ -9,4 +9,10 @@
             <script type="text/javascript" src="/purchase/static/src/js/tours/purchase.js"></script>
         </xpath>
     </template>
+
+    <!-- <template id="assets_frontend" name="purchase assets" inherit_id="web.assets_frontend">
+        <xpath expr="." position="inside">
+            <script type="text/javascript" src="/purchase/static/src/js/purchase_portal_datetimepicker.js"></script>
+        </xpath>
+    </template> -->
```

Comment:
to clean

Reply:
I have a small issue with datetimepicker on the portal, it can't properly toggle when lose focus. Website people told me I need to instantiate my datetimepicker in JS. But currently I still have trouble with doing this.